pydantic_ai_slim/pydantic_ai/result.py

```diff
@@ -116,7 +116,7 @@ async def stream_text(self, *, delta: bool = False, debounce_by: float | None =
         else:
             async for text in self._stream_response_text(delta=False, debounce_by=debounce_by):
                 for validator in self._output_validators:
-                    text = await validator.validate(text, self._run_ctx)  # pragma: no cover
+                    text = await validator.validate(text, self._run_ctx, partial=True)
```

Comment:
My suggestion above would mean passing the value through `RunContext` in this case as well... I kinda like having it as a separate arg, but that's harder to justify if we require the `ctx` arg and are in some cases passing it through there anyway